planet/src/main/webapp/places.js

```diff
@@ -241,98 +261,94 @@ function createMarkers(place) {
     });
 }
 
-/** List results of nearby search on page*/
-function listResults() {
+/** List result of searched place on page*/
+function listResults(placeInfo) {
```

Comment:
nit: this function's name no longer matches its behaviour. It's pluralized, but it's handling a single result.

Maybe `renderResult`?